database/changesets/20200616_AllOfUsArraysTable.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="chengche@broadinstitute.org"
+               dbms="postgresql"
+               id="20200616"
+               logicalFilePath="20200616_AllOfUsArraysTable.xml">
+        <comment>
+            the AllOfUsArrays table schema
+        </comment>
+        <sql>
+            CREATE TYPE AllOfUsArrays AS (
+                analysis_version_number     bigint,      -- input to the workflow (PK part1)
+                chip_well_barcode           text,        -- input to the workflow (PK part2)
```

Comment:
@pshapiro4broad @gbggrant Is it safe to make the assumption that `(analysis_version_number, chip_well_barcode)` pairs will always be unique, unless we intentionally want to re-run a sample with a specific version?

Reply:
@rexwangcc - yes, that's a safe assumption to make.  You really woudn't re-run a sample with a specific version in production.

Reply:
Thanks!!